src/finch/tensor.py

```diff
@@ -1,5 +1,6 @@
 import builtins
 from typing import Any, Callable, Optional, Iterable, Literal
+import warnings
```

Comment:
We don't use `warnings` anymore do we?

Reply:
There's a warning in `def reshape`, so we need to keep that import.